R/get_smr.R

```diff
@@ -1,96 +1,162 @@
-#' Perform indirect rate standardization
+#' Calculate a standardized morbidity/mortality ratio
 #'
-#' @description Compute a standardized mortality ratio (SMR) or expected number
-#' of deaths given the number of observed deaths, the size of each stratum in
-#' the observed population, and the specific rate for each stratum in the
-#' standard population.
+#' @description Calculate a standardized morbidity/mortality ratio (SMR) or the
```

Comment:
Might be worth defining the SMR as the number of observed events / expected events in this description (or if too long, then add to details)